recipes/umi-transfer/meta.yaml

```diff
@@ -11,20 +11,16 @@ source:
   
 build:
   number: 0
+  skip: true
+  run_exports:
+    - {{ pin_subpackage('umi-transfer', max_pin="x") }}
 
 requirements:
   build:
-    - rust >=1.40
-    - cmake
-    - make
-    - autoconf
-    - {{ compiler('cxx') }}
-    - pkg-config
-    - zlib
-  host:
-    - zlib
-  run:
-
+    - {{ compiler("rust") }}
+    - rust
+    - cargo
```

Comment:
No need to list `rust` and `cargo`. They come with `{{ compiler("rust") }}`

Reply:
Thanks for the valuable tip! I had been looking [for some guidance on Rust tools here](https://bioconda.github.io/contributor/guidelines.html), but apparently it is still too irrelevant for an own section.